javascript/tree-js/index.js

```diff
@@ -1,20 +1,19 @@
-function tree(nameItemsObj, indent) {
+function tree(nameItemsObj, indent = '') {
   for (const [key, value] of Object.entries(nameItemsObj)) {
     // Print out name
     if (key === 'name') {
       console.log(indent, value)
     }
 
-    // Indent items
+    // Check and structure items
     if ((key === 'items') & Array.isArray(value)) {
-      // let indentItem = ' ├——',
-      //   indentChild = ' │   └——'
       let indentItem = ' \u251c\u2014\u2014',
-        indentChild = ' \u2502   \u2514\u2014\u2014'
-      indent = indent === indentItem ? indentChild : indentItem
+        indentChild = ' \u2502  '
```

Comment:
Not for fixing, rather a general advice.
It really helps in code like this to give names for constants like indentation.
F.e. `const INDENT_VERTICAL = ' \u2502  '`

helps to operate with the code and think on logic, not on small details